.github/workflows/app-distribute.yml

```diff
@@ -30,11 +30,11 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: dogfooding-release
-          path: dogfooding/build/outputs/apk/release/
+          path: dogfooding/build/outputs/apk/dogfooding/release/
       - name: Upload artifact to Firebase App Distribution
         uses: wzieba/Firebase-Distribution-Github-Action@v1.5.0
         with:
           appId: ${{secrets.FIREBASE_DOGFOODING_SAMPLE_APP_ID}}
           serviceCredentialsFileContent: ${{ secrets.CREDENTIAL_FILE_CONTENT }}
           groups: stream-testers
-          file: dogfooding/build/outputs/apk/release/dogfooding-release.apk
\ No newline at end of file
+          file: dogfooding/build/outputs/apk/dogfooding/release/dogfooding-dogfooding-release.apk
```

Comment:
It this an intentional doubling of the `dogfooding` word in the file name? :)

Reply:
As far as we are using `dogfooding`/`production` flavors and the project is named `dogfooding` as well, it is the result name used for the output apk.
We will change the project name in the future